src/widgets/medications/medications-overview.component.tsx

```diff
@@ -198,6 +171,128 @@ export default function MedicationsOverview(props: MedicationsOverviewProps) {
+        <SummaryCard
+          name={t("Active Medications", "Active Medications")}
+          styles={{ width: "100%" }}
+          link={`${props.basePath}`}
+          showComponent={() =>
+            openWorkspaceTab(MedicationOrderBasket, "Medication Order")
+          }
+        >
+          <SummaryCardRow>
+            <SummaryCardRowContent>
+              <HorizontalLabelValue
+                label="Active Medications"
+                labelStyles={{
+                  color: "var(--omrs-color-ink-medium-contrast)",
+                  fontFamily: "Work Sans"
+                }}
+                value=" "
+                valueStyles={{
+                  color: "var(--omrs-color-ink-medium-contrast)",
+                  fontFamily: "Work Sans"
+                }}
+              />
+            </SummaryCardRowContent>
+          </SummaryCardRow>
+          <table className={styles.medicationsTable}>
+            <tbody>
+              {patientMedications.map((medication, index) => (
+                <tr key={index}>
+                  <td>
+                    <span
+                      style={{
+                        fontWeight: 500,
+                        color: "var(--omrs-color-ink-high-contrast)"
+                      }}
+                    >
+                      {medication.drug.name}
+                    </span>{" "}
+                    &mdash;{" "}
+                    <span
+                      style={{ color: "var(--omrs-color-ink-medium-contrast)" }}
+                    >
+                      {" "}
+                      DOSE
+                    </span>{" "}
+                    <span
+                      style={{
+                        fontWeight: 500,
+                        color: "var(--omrs-color-ink-high-contrast)"
+                      }}
+                    >
+                      {getDosage(
+                        medication?.drug?.strength,
+                        medication?.dose
+                      ).toLowerCase()}
+                    </span>{" "}
+                    &mdash;
+                    <span>
+                      {" "}
+                      {(medication?.doseUnits?.display).toLowerCase()}
+                    </span>{" "}
+                    &mdash;{" "}
+                    <span>{(medication?.route?.display).toLowerCase()}</span>{" "}
+                    <span
+                      style={{
+                        fontWeight: 400,
+                        color: "var(--omrs-color-ink-high-contrast)"
+                      }}
+                    >
+                      &mdash; {medication.frequency.display}
+                    </span>
+                  </td>
+                  <td>
+                    <MedicationButton
+                      component={MedicationOrderBasket}
+                      name={"Medication Order Basket"}
+                      label={"Revise"}
+                      orderUuid={medication.uuid}
```

Comment:
From here on there are no conditional members - maybe `medication` is always *not* `undefined`?

Reply:
Sure. Good spot.